README.md

```diff
@@ -133,6 +133,24 @@ Run your app in production mode:
 npm run prod
 ```
 
+Run your app with an attached HTML Validator (HTML Validator is attached by default):
+
+```
+npm run dev attach
+```
+
+Run your app with a detached HTML Validator:
```

Comment:
Run your app with a detached HTML validator (runs validator as a separate process in the background; process must be manually killed later, see below):